src/components/controls/dropdown/Dropdown.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { noop } from 'lodash';
```

Comment:
If noop is the only thing we're using lodash for, could we not just use "()=>{}" instead, and get rid of the dependency?